src/dataService/savedItemsService.ts

```diff
@@ -297,8 +296,8 @@ export class SavedItemDataService {
         try {
           await this.db(table)
             .delete()
-            .where('item_id', id)
-            .andWhere({ user_id: this.userId });
+            .where({ user_id: this.userId })
```

Comment:
this ordering doesn't matter according to documentation as long as both these fields are in the same composite index.